src/main/resources/com/cloudbees/jenkins/plugins/amazonecs/pipeline/ECSTaskTemplateStep/config.jelly

```diff
@@ -261,5 +261,22 @@
         </table>
       </f:repeatable>
     </f:entry>
+    <f:entry field="placementStrategies" title="${%Placement Strategy}">
+      <f:repeatable field="placementStrategies">
+        <table width="100%">
+          <f:entry field="type" title="${%Placement Type}">
```

Comment:
I feel like this should be a dropdown with the different placement strategy types. (Spread, Random, BinPack)

Also, some documentation on this would also help. Sourced from AWS documentation?